benches/bench.rs

```diff
@@ -217,7 +217,7 @@ fn access_column(b: &mut Bencher) {
 
     b.iter(|| {
         for i in 0..100 {
-            let comp = column.get(entc).unwrap();
+            let comp = bencher::black_box(column.get(entc).unwrap());
```

Comment:
I don't even know if this works now ? or how to test it

Reply:
Only way to know for sure is to look at the output assembly or IR, but at least we're making an effort now. Please also resolve the lint.